packages/node_modules/@ciscospark/react-component-utils/src/adaptiveCardUtil.js

```diff
@@ -52,7 +71,6 @@ export function getAdaptiveCard(cards, displayName, sparkInstance) {
  * @param {object} sparkInstance
  * @returns {boolean}
  */
-
 export function hasAdaptiveCard(cards, sparkInstance) {
   if (cards.length > 0 && Array.isArray(cards)) {
```

Comment:
these two check statements should be swapped.

Reply:
swapped the two statements